index.html

```diff
@@ -0,0 +1,244 @@
+  right: 0;
+  bottom: 0;
+  background: linear-gradient( white, grey, grey);
+  z-index: -2;
+  filter: blur(20px);
+}
+
+label {
+  color: black;
+  font-size: 13px;
+  padding-bottom: 10px;
+  font-weight: bold;
+  font-family: 'Raleway';
+}
+
+.btn {
+  padding: 10px;
+  width: 25%;
+  margin-right: 5px;
+  margin-top: 10px; 
+  font-family: 'Raleway';
+  border: 2px solid #EFEFEF;
+  background-color: white;
+  font-weight: bold;
+  color: grey !important;
+}
+
+.startbtn {
+  background-color: rgb(30, 172, 178);
+  padding: 15px;
+  color: white!important;
+  width: 80%;
+  margin-left: 33px;
+  border: none;
+  border-radius: 6px;
+  font-size: 16px;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+}
+
+.callus {
+  margin-left: 93px;
+  font-weight: bold;
+  font-size: 12px;
+  color: grey;
+  margin-top: 25px;
+  font-family: 'Roboto';
+}
+</style>
+</head>
+<title>Excercise2</title>
+<body>
+  <div class="outer">
+  <div  class="dev">
+    <h3 class="heading">Why Loconav's Vehicle Tracking System ?</h3>
+    <div>
+      <div class="row">
+        <div class="column" style="background-color: #FFF">
+          <img src="image1.png" class="vehicle-track-image">
+          <h4 class="headinternal">INTELLIGENT VEHICLE TRACKING</h4>
+          <p class="description"> Wheather you are single vehicle or a <span style="color: #6399C7">fleet owner</span>, Loconav is determined to protect and monitor your asset from theft or harm</span></p>
+        </div>
+        <div class="column" style="background-color: #FFF" >
+          <img src="image4.png" class="vehicle-lock-image" >
+          <h4 class="headinternal">
+          VEHICLE LOCK</h3>
+          <p class="description"> Turn on vehicle lock from Loconav's <span style="color: #6399C7">mobile app</span> and be assured that your vehicle won't start without your permission</p>
+        </div>
+        <div class="column" style="background-color: #FFF">
+          <img src="image2.png" class="live-alert-image" >
+          <h4 class="headinternal">
+          LIVE ALERTS</h3>
+          <p class="description"> Get live over-speeding alerts,alerts on entry and exit points,monitor idling in real time ,etc. Vehiccle services and maintenance alerts</p>
+        </div>
+        <div class="column" style="background-color: #FFF" >
+          <img src="image3.png" class="vehicle-health-image" >
+          <h4 class="headinternal">
+          VEHICLE HEALTH</h3>
+          <p class="description" > Get diagostic,mileage and health report of your vehicle so that you can worry less about your vehicle's well-being and enjoy your time on road</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="loginform">
+    <h3 class ="tagline"> Get an offer on your mobile</h3>
+    <input type="text" name="phonenumber" placeholder="Phone Number" class="inputd">
+    <input type="number" name="noofvehicle" placeholder="How many vehicles ?" class="inputd">
+    <div style="margin-left: 33px">

```

Comment:
please use your css instead of defining style explicitly for this div

Reply:
having problem in this